src/config/__tests__/settings.spec.ts

```diff
@@ -0,0 +1,228 @@
+import fs from 'fs';
+
+import * as settings from '../settings';
+import { PathManagementStrategy } from '@/integrations/pathManager';
+
+describe('updateFromCommandLine', () => {
+  let prefs: settings.Settings;
+  let origPrefs: settings.Settings;
+
+  beforeEach(() => {
+    jest.spyOn(fs, 'writeFileSync').mockImplementation(() => { });
+    prefs = {
```

Comment:
This will break if we add more fields to `Settings`.
I guess that will fail the test and we can fix it then? Though it might also work as
```ts
prefs = JSON.parse(JSON.stringify(settings.defaultSettings));
prefs.kubernetes.Version = '1.23.5';
// and any more fixtures we want  here
```
(Or use `_.defaults` or whatever)

Reply:
I want *some* test to fail when the settings object is changed. No other tests do that currently, so we might as well make that check here.